e2e/404.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { test, expect, Page } from '@playwright/test';
+
+test.describe('404 Page', () => {
+  let page: Page;
+
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+    await page.goto('/404');
+  });
+
+  test.afterAll(async () => {
+    await page.close();
+  });
```

Comment:
```suggestion
  test.beforeEach(async ({ page }) => {
    await page.goto('/settings');
  });
```
Ref: https://github.com/freeCodeCamp/freeCodeCamp/pull/51754#discussion_r1342543051